src/user.clj

```diff
@@ -0,0 +1,8 @@
+(ns user)
+
+(defn dev
+  "Load and switch to the 'dev' namespace."
+  []
+  (require 'dev)
+  (in-ns 'dev)
+  :loaded)
```

Comment:
can't we put this under `dev/src` as well? So it doesn't get included in the uberjar